src/pyOpenMS/tests/test_ProForma.py

```
@@ -0,0 +1,308 @@
+    # Unlocalised modification should have issues
+    pf = p.parse("[Phospho]?PEPTIDE")
+    issues = p.getAASequenceConversionIssues(pf)
+    assert len(issues) > 0
+
+
+def test_parse_error_handling():
+    """Test that parse errors are properly raised."""
+    import pyopenms as p
+
+    # Unclosed bracket should fail
+    with pytest.raises(Exception):
```

Ruff (0.14.13)

Do not assert blind exception: `Exception`

(B017)